src/main.styl

```diff
@@ -8,7 +8,7 @@ $venetian-red = #D0021B
 $white = #FFFFFF
 
 body
-  font-size 14px
+  font-size 100%
```

Comment:
@eleanormal you specifically didn't want this, correct?

Reply:
I thought we were supposed to use rem

Reply:
We shouldn't use 100% especially on the body because the body size is what sets the rest. Most browsers have 16px as their default font-size, so setting it to 100% makes that the base font size. If most of our fonts will be 14 px, then we should set that as our base font-size and then other places where we have text should be set using rem